replicate/common/common.go

```diff
@@ -42,3 +43,25 @@ func BuildStrictRegex(regex string) string {
 func JSONPatchPathEscape(annotation string) string {
 	return strings.ReplaceAll(annotation, "/", "~1")
 }
+
+type Annotatable interface {
+	GetAnnotations() map[string]string
+	SetAnnotations(map[string]string)
+}
+
+func CopyAnnotations[I, O Annotatable](input I, output O) {
+	val := input.GetAnnotations()
+	copy := make(map[string]string, len(val))
+
+	strip, ok := val[StripAnnotations]
+	if !ok || strings.ToLower(strip) != "true" {
```

Comment:
Not sure what unexpected effects it could have to make copying the labels the default. Maybe it is safer to have to explicitly enable the copying?

Reply:
Just thought of another argument in favor switching the default behavior: If this PR was merged, and the replicator is updated in a cluster, nothing will happen yet ([because](https://github.com/mittwald/kubernetes-replicator/issues/349)). The labels will only be copied for resources that are updated after the new replicator is deployed. That means the behavior would be more consistent if users have to specifically activate the behavior on resources where they need it.